### DISCUSSION TITLE
the specific meaning of the human's limb poses in the bathing scenario

### DISCUSSION THREAD

**Discussion by TomatoPicker278** [empriselab/RCareWorld#162] [Other Q&A] [answered]
On the official website of PhyRoboCare, it states:

“What aspects of each task are randomized?
For bathing: the human's limb poses, the initial position of the sponge, and the initial position of the robot's base.”

I would like to ask about the specific meaning of "human's limb poses" here. Does "limb poses" refer specifically to the poses of the two arms? Or does it include the poses of both arms and legs? Or does it include the poses of both arms, legs, and the body? Thank you.

**Comment by YoruCathy** [comment, marked as answer]
It means both the arms and the legs. If you run the scene multiple times, you will see the pose changes of the human limbs.

**Comment by TomatoPicker278** [comment]
Thank you for your response. I understand what you mean — that the poses of the arms and legs are randomized, but the pose of the body are fixed.
